jenkins/Jenkinsfile.release

```diff
@@ -57,7 +57,8 @@ pipeline {
                     sh "mkdir -p ${HOME}/.zinc"
                     docker.image("$IMAGE_NAME").inside("--runtime=nvidia -v ${HOME}/.m2:${HOME}/.m2:rw \
                         -v ${HOME}/.zinc:${HOME}/.zinc:rw") {
-                        sh "mvn -U -B clean install $MVN_MIRROR -P 'include-databricks,source-javadoc,!snapshot-shims'"
+                        sh "mvn dependency:purge-local-repository -DmanualInclude='com.nvidia:rapids-4-spark-shims-spark300-databricks_2.12' \
```

Comment:
What's the intent of this modification ?

Reply:
same question

Reply:
Because we have released the v0.2.0 jars several times. And for the release version of jars, if it already exists in the local `.m2`